metadata.json

```diff
@@ -1,15 +1,21 @@
 {
   "name": "puppet-misp",
-  "version": "0.2.0",
+  "version": "0.2.3-rc0",
   "author": "Vox Pupuli",
   "summary": "Module to install MISP (Malware Information Sharing Platform)",
   "license": "Apache-2.0",
   "source": "https://github.com/voxpupuli/puppet-misp",
   "project_page": "https://github.com/voxpupuli/puppet-misp",
   "issues_url": "https://github.com/voxpupuli/puppet-misp/issues",
   "dependencies": [
-    {"name":"puppetlabs-stdlib","version_requirement":">= 1.0.0 < 5.0.0"},
-    {"name":"puppetlabs-vcsrepo","version_requirement":">= 1.3.2 < 2.0.0"}
+    {
+      "name": "puppetlabs-stdlib",
+      "version_requirement": ">= 1.0.0 < 5.0.0"
```

Comment:
please bump the 1.0.0 to 4.13.1. That is the first version that works properly with puppet 4